### PR TITLE
Code reformat

### PR DESCRIPTION
**Pull request by valdisiljuconoks**
Added license header info

### CODE REVIEW

File: src/Geta.GoogleProductFeed/Controllers/GoogleProductFeedController.cs [hunk 1]
```diff
@@ -1,4 +1,8 @@
-﻿using System.Linq;
+﻿// Copyright (c) Geta Digital. All rights reserved.
+// Licensed under MIT.
+// See the LICENSE file in the project root for more information
```
Comment (frederikvig) [inline src/Geta.GoogleProductFeed/Controllers/GoogleProductFeedController.cs]: I would put it on two lines instead (Geta or Geta Digital, it says just Geta in nuspec):
// Copyright (c) Geta Digital. All rights reserved.
// Licensed under MIT. See the LICENSE file in the project root for more information.

File: src/Geta.GoogleProductFeed/Controllers/GoogleProductFeedController.cs [hunk 2]
```diff
@@ -21,7 +24,7 @@ public IHttpActionResult Get()
             var siteHost = HttpContext.Current.Request.Url.Host;
             var feed = _feedHelper.GetLatestFeed(siteHost);
 
-            if (feed == null)
+            if(feed == null)
```
Comment (frederikvig) [inline src/Geta.GoogleProductFeed/Controllers/GoogleProductFeedController.cs]: There should be a space between

File: src/Geta.GoogleProductFeed/FeedHelper.cs [hunk 3]
```diff
@@ -23,9 +26,9 @@ public FeedHelper(FeedBuilder feedBuilder, IFeedRepository feedRepository)
 
         public bool GenerateAndSaveData()
         {
-            List<Feed> feeds = _feedBuilder.Build();
+            var feeds = _feedBuilder.Build();
```
Comment (frederikvig) [inline src/Geta.GoogleProductFeed/FeedHelper.cs]: It's good to see the return type when reading the code

File: src/Geta.GoogleProductFeed/Controllers/GoogleProductFeedController.cs (hunk 2 above)
Comment (valdisiljuconoks) [reply src/Geta.GoogleProductFeed/Controllers/GoogleProductFeedController.cs]: why?

Comment (frederikvig) [reply src/Geta.GoogleProductFeed/Controllers/GoogleProductFeedController.cs]: Consistency and readability. Basically  we've been following the guidelines from Microsoft .NET team on this. 

Example: 
image

**Pull request merged**